spec/system/private_message_spec.rb

```diff
@@ -44,5 +44,20 @@
       composer.open_composer_actions
       expect(composer).to have_no_action("New message")
     end
+
+    it "can switch between topic and personal message modes" do
+      visit "/new-topic"
+      expect(composer).to be_opened
+
+      # Switch to personal message
+      composer.open_composer_actions
+      composer.select_action("New message")
```

Comment:
Can you switch these action labels + the ones in other specs to use I18n please instead of the hardcoded strings?